docs/chi-examples/20-protobuf-schema-example.yaml

```diff
@@ -0,0 +1,70 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: protobuf-schemas
+data:
+  your_schema.proto: |
+    syntax = "proto3";
+    package tutorial;
+    import "google/protobuf/timestamp.proto";
+
+    message Person {
+      string name = 1;
+      int32 id = 2;  // Unique ID number for this person.
+      string email = 3;
+
+      enum PhoneType {
+        MOBILE = 0;
+        HOME = 1;
+        WORK = 2;
+      }
+
+      message PhoneNumber {
+        string number = 1;
+        PhoneType type = 2;
+      }
+
+      repeated PhoneNumber phones = 4;
+
+      google.protobuf.Timestamp last_updated = 5;
+    }
+
+---
+apiVersion: "clickhouse.altinity.com/v1"
+kind: "ClickHouseInstallation"
+metadata:
+  name: "protobuf-example"
+spec:
+  configuration:
+    settings:
+      format_schema_path: /opt/format_schemas/
+    clusters:
+      - name: cluster
+        layout:
+          shardsCount: 1
+          replicasCount: 1
+
+  defaults:
+    templates:
+      podTemplate: pod-template-with-volumes
+
+
+  templates:
+    podTemplates:
+      - name: pod-template-with-volumes
+        spec:
+          securityContext:
```

Comment:
@sunsingerus @alex-zaitsev could share your mention?

we have conflict here
- configMap always mount as root:root owner and as readOnly file system
- `clickhouse-server` always try to change ownership for `format_schema_path` and failed to start